### ISSUE TITLE
The unauthenticated git protocol on port 9418 is no longer supported

### ISSUE THREAD

**Issue by pcotret** [chipsalliance/rocket-tools#20]
```bash
$ git clone https://github.com/freechipsproject/rocket-tools
$ cd rocket-tools
$ git submodule update --init --recursive
```
Got me into the error described in the title.
In other words, recursive clone of the RISC-V GNU toolchain can't work since March 15:
https://github.com/riscv-collab/riscv-gnu-toolchain/blob/64879b24e18572a3d67aa4268477946ddb248006/.gitmodules#L18
`git://github.com/riscv/riscv-qemu.git` isn't a valid URL anymore as it waits for `https://git.qemu.org/git/qemu.git`

This issue has been fixed in July 2019: https://github.com/riscv-collab/riscv-gnu-toolchain/commit/80fd8519bb7d1a918bd8a7852ba5adb94ea65c57

Not sure if we could use this version of [riscv-gnu-toolchain](https://github.com/riscv-collab/riscv-gnu-toolchain/tree/80fd8519bb7d1a918bd8a7852ba5adb94ea65c57) with other submodules of ̀`rocket-tools`.

## Reference
https://github.blog/2021-09-01-improving-git-protocol-security-github/
Section Git protocol troubleshooting
> If any of them start with `git://`, you should [change the URL](https://docs.github.com/en/get-started/getting-started-with-git/managing-remote-repositories#changing-a-remote-repositorys-url) to a supported format.

**Comment by pcotret** [comment]
Found a solution thanks to @QDucasse. This way, we can still clone the version given in the [rocket-chip repository](https://github.com/chipsalliance/rocket-chip/blob/master/riscv-tools.hash):
```bash
git clone https://github.com/chipsalliance/rocket-tools
cd rocket-tools
# Hash given in the rocket-chip repository
git checkout e2c6d1577a75f506fe992c3eb20a75174504476e
git submodule update --init riscv-gnu-toolchain
cd riscv-gnu-toolchain
git submodule set-url riscv-qemu https://github.com/riscv/riscv-qemu.git
git submodule sync
cd riscv-qemu
git submodule set-url roms/qemu-palcode https://github.com/rth7680/qemu-palcode.git
git submodule sync
cd ../..
git submodule update --init --recursive --progress
```

**Comment by ZenithalHourlyRate** [comment]
Fixed by c17f76bfa09567f484f504825ec677fe15e12143
